_upgrade-to/upgrade-to.md

```diff
@@ -179,3 +179,97 @@ If you are upgrading an Open Distro for Elasticsearch cluster, we recommend firs
      }
    }
    ```
+
+## Upgrade tool
+
+The `opensearch-upgrade` tool lets you automate some of the steps in [Upgrade to OpenSearch]({{site.url}}{{site.baseurl}}/upgrade-to/upgrade-to/#upgrade-to-opensearch), eliminating the need for error-prone manual operations.
+
+The `opensearch-upgrade` tool performs the following functions:
+
+- Imports any existing configurations and applies it to the new installation of OpenSearch.
+- Installs any existing core plugins.
+
+### Limitations
+
+The `opensearch-upgrade` tool doesn't perform an end-to-end upgrade:
+
+- You need to run the tool on each node of the cluster individually as part of the upgrade process.
+- The tool doesn't provide a rollback option after you've upgraded a node, so make sure you follow best practices and take backups.
+- You must install all community plugins (if available) manually.
+- The tool only validates any keystore settings at service start-up time, so you must manually remove any unsupported settings for the service to start.
+
+### Using the upgrade tool
+
+To perform a rolling upgrade using the [OpenSearch tarball]({{site.url}}{{site.baseurl}}/opensearch/install/tar/) distribution:
+
+Check [Upgrade paths]({{site.url}}{{site.baseurl}}/upgrade-to/upgrade-to/#upgrade-paths) to make sure that the version you’re upgrading to is supported and whether you need to upgrade to a supported Elasticsearch OSS version first.
+{: .note }
+
+1. Disable shard allocation to prevent Elasticsearch OSS from replicating shards as you shut down nodes:
+
+   ```json
+   PUT _cluster/settings
+   {
+     "persistent": {
+       "cluster.routing.allocation.enable": "primaries"
+     }
+   }
+   ```
+
+1. On any one of the nodes, download and extract the OpenSearch tarball to a new directory.
+
+1. Make sure the following environment variables are set:
```

Comment:
I think it's worth including an example export command here for how to set one of these.

Reply:
Added export commands.